dexmaker/src/test/java/com/google/dexmaker/DexMakerTest.java

```diff
@@ -66,6 +67,15 @@
     private void reset() {
         dexMaker = new DexMaker();
         dexMaker.declare(GENERATED, "Generated.java", PUBLIC, TypeId.OBJECT);
+        clearDataDirectory();
+    }
+
+    private void clearDataDirectory() {
```

Comment:
This is clearing the dexCache, right?
Shouldn't this be doing a recursive delete since we're nesting directories for each DexMaker version?